File: TaxiGestion/Dto/TGC/DtoTGC001OutDC21EcritureForListColl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaxiGestion.Dto.TGC
{
    public class DtoTGC001OutDC21EcritureForListColl
    {
        public int NoEcriture { get; set; }
        public int? NoCompteDebit { get; set; }
        public string DesiCompteDebit { get; set; }
        public int? NoCompteCredit { get; set; }
        public string DesiCompteCredit { get; set; }
        public DateTime DateEcriture { get; set; }
        public int NoPiece { get; set; }
        public double Debit { get; set; }
        public double Credit { get; set; }
        public double Solde { get; set; }
        public string Libelle1 { get; set; }
        public string Libelle2 { get; set; }
        public bool SwImpressionExtourne { get; set; }
    }
}

File: TaxiGestion/ClientApp/src/app/_dto/TGC/DtoTGC001OutDC21EcritureForListColl.ts
export interface DtoTGC001OutDC21EcritureForListColl {
    noEcriture: number;
    noCompteDebit?: number;
    desiCompteDebit: string;
    noCompteCredit?: number;
    desiCompteCredit: string;
    dateEcriture: Date;
    noPiece: number;
    debit: number;
    credit: number;
    solde: number;
    libelle1: string;
    libelle2: string;
    swImpressionExtourne: boolean;
}